src/electionguard-ui/ElectionGuard.UI/MauiProgram.cs

```diff
@@ -126,5 +131,29 @@ public static MauiAppBuilder SetupServices(this MauiAppBuilder builder)
         return builder;
     }
 
+    private static void InitializeAppCenter()
+    {
+
+#if !DEBUG
+#if APPCENTER_SECRET_MACOS
+        string macSecret = APPCENTER_SECRET_MACOS;
+#else
+        string macSecret = null;
+#endif
 
+#if APPCENTER_SECRET_UWP
+        string uwpSecret = APPCENTER_SECRET_UWP;
+#else
+        string uwpSecret = null;
+#endif
+        if (!string.IsNullOrWhiteSpace(macSecret) || !string.IsNullOrWhiteSpace(uwpSecret))
+        {
+            AppCenter.Start(
+                $"uwp={uwpSecret};" +
+                $"macos={macSecret};",
+                typeof(Crashes));
+        }
```

Comment:
Does the call to AppCenter.Start like it when one of the fields is null?  would it be better to use StringBuilder to create that string based on the data that we have?

What happens if we run a local version in release mode without either setting setup??

Reply:
This should be a template string: `($"uwp={uwpSecret};macos={macSecret}", typeof(Crashes))`